static/app/components/group/suggestedOwners/ownershipRules.tsx

```diff
@@ -78,3 +151,39 @@ const HelpfulBody = styled('div')`
   padding: ${space(1)};
   text-align: center;
 `;
+
+const Container = styled(Panel)`
+  background: none;
+  display: flex;
+  flex-direction: column;
+  padding: ${space(2)};
+`;
+
+const HeaderContainer = styled('div')`
+  display: flex;
+  align-items: flex-start;
+`;
+
+const Header = styled('h4')`
+  margin-bottom: ${space(1)};
+  text-transform: uppercase;
+  font-weight: bold;
+  color: ${p => p.theme.gray400};
+  font-size: ${p => p.theme.fontSizeMedium};
+`;
+
+const Content = styled('span')`
+  color: ${p => p.theme.subText};
+  margin-bottom: ${space(2)};
+`;
+
+const SetupButton = styled(Button)`
+  &:focus {
+    color: ${p => p.theme.white};
+  }
+`;
+
+const DismissButton = styled(Button)`
+  margin-left: auto;
```

Comment:
you can probably get rid of the margin-left here if you use `justify-content: space-between` up in the `HeaderContainer` which is probably marginally more correct